utility_test.go

```diff
@@ -169,7 +171,7 @@ func makeOrderFinalised(t *testing.T, supportedChalTypes []string, identifiers .
 		chalType := supportedChalTypes[mrand.Intn(len(supportedChalTypes))]
 		chal, ok := auth.ChallengeMap[chalType]
 		if !ok {
-			t.Fatalf("No supported challenge %q (%v) in challenges: %v", chalType, supportedChalTypes, auth.ChallengeTypes)
+			t.Skipf("skipping, no supported challenge %q (%v) in challenges: %v", chalType, supportedChalTypes, auth.ChallengeTypes)
```

Comment:
This change of a fatal to skip is due to the unconditional execution of `TestWildcardDNSAccount` when the testing against a Boulder/Pebble instance without `dns-account-01` support. There may be a preferable way to accomplish this without making this a skip.